code/magento-os/lightna-theme/Magento_Theme/web/css/lightna.css

```diff
@@ -0,0 +1,1822 @@
+*,:after,:before {
```

Comment:
is this file also copied from lightna without changes?

Reply:
nope, unfortunately

Reply:
I guess we need one more round of discussion on that topic

Reply:
ok, probably we can include lightna styles + addl small lightna-semi.css

Reply:
probably even separate lightna module magento-os-frontend-semi

Reply:
basically, file incudes only lightna styles BUT I needed to change rem to px and correct images url => these are the changes I meant

Reply:
probably we can solve it, but yeah, lets have a call

Reply:
I can resolve rem to px thing, but need to thing about images/fonts